webapp/src/ts/services/lineage-model-generator.service.ts

```diff
@@ -112,6 +112,14 @@ export class LineageModelGeneratorService {
         });
       });
   }
+
+  docs(docs) {
+    return this.ngZone.runOutsideAngular(() => this._docs(docs));
+  }
+
+  private _docs(docs) {
+    return this.lineageLib.hydrateDocs(docs);
+  }
```

Comment:
Same here...

Reply:
Strange, for some reason the UI doesn't get the "Outdated" label, even though the code has changed: https://github.com/medic/cht-core/pull/7088/files#diff-b3841c4f18f0a7052a516aad6cf4af50bbb1c6c90ff14321903bd55a53a718b5